kuma/settings/common.py

```diff
@@ -1831,6 +1847,8 @@ def get_user_url(user):
 # Whether or not to enable the BCD signalling feature.
 # Affects loading of CSS (statically) and JS (in runtime).
 ENABLE_BCD_SIGNAL = config('ENABLE_BCD_SIGNAL', default=True, cast=bool)
+# Multi Auth Switch
```

Comment:
That comment isn't adding any value.

Reply:
I added it more as a quick way to find the switch but, the name of the property is just as easy to search for :) I will add a bit more context around it.